src/Avalonia.Base/AvaloniaObject.cs

```diff
@@ -332,6 +335,7 @@ public bool IsSet(AvaloniaProperty property)
                 }
 
                 subscription = source
+                    .Do(_ => VerifyAccess())
```

Comment:
Yeah, I wasn't sure whether to keep this in or not: if someone replaces the scheduler then it's their fault really. I mainly added it for completion's sake in terms of unit tests, but I'm torn on whether to keep it,